main.go

```diff
@@ -117,11 +118,11 @@ func main() {
 						log.Warnf("failed to get build artifact, error: %s", err)
 					}
 
-					downloadErr := artifactObj.Artifact.DownloadArtifact(strings.TrimSpace(cfg.BuildArtifactsSavePath) + artifactObj.Artifact.Title)
+					downloadErr := artifactObj.Artifact.DownloadArtifact(strings.TrimSpace(fullBuildArtifactsSavePath) + artifactObj.Artifact.Title)
```

Comment:
go standard lib has the functionality to join path elements: [filepath.Join()](https://golang.org/pkg/path/filepath/#Join), I suggest using it.

By using `filepath.Join()` you do not need to care about trailing path separators.